scripts/dashboard/medperf_dashboard/assets/stages.csv

```diff
@@ -0,0 +1,51 @@
+
+``` tar -xzf review_cases.tar.gz ```
+
+If working remotely, move the tarball file to your local machine. Untar the tarball file, select the best baseline segmentation and make any necessary corrections.  Once you're done, move the finalized file to the `finalized` folder, retaining the original name.
+
+If using the monitoring tool, create a new tarball with the previously untarred files, as well as finalized cases. You can do so with the following command
+
+``` tar -czf reviewed_cases.tar.gz -C review_cases . ```
+
+Place this new tarball file on the directory the monitoring tool is running (displayed on the monitoring tool header). The tool will automatically detect the finalized cases and place them in the expected locations.
+
+### Brain Mask Correction
+There might be situations where the brain mask is not correct. If working locally you may use the `Review Brain Mask` button to automatically make brain mask corrections with ITK-Snap. If instead you're using the previously menitoned tarball, you may find the brain mask segmentation as `brainMask_fused.nii.gz`. Please make the necessary corrections under that file, overwriting the contents. Once the new tarball is created and placed within the directory the monitoring tool is running, it will be automatically identified and placed in the correct location.
+**NOTE** Modifying the brain mask invalidates the tumor segmentations, and re-runs the tumor segmentation procedure using this new mask.
+",,mediumseagreen
+-5.1,Multiple Annotations Error,More than one segmentation file was identified in the manually corrected output folder. Please ensure that there is only one manually corrected segmentation inside the labels path.,,mediumpurple
+-6,Comparison Complete,,,mediumturquoise
+-6.1,Exact Match Identified,"The automated and manually corrected segmentation files are identical. Was this intentional? If so, are you certain that no manual corrections were required? If not, please redo manual correction for this exam.",,orchid
+-6.2,Annotation Comparison Failed,The original segmentation for the reviewed file was not identified. This most probably means the annotation file was renamed. Please ensure the reviewed file retains its original name.,,peachpuff
+7,Annotation Confirmed,,,silver
+8,Done,,,green
+-0.101,Generate Report Unhandled Error,,,tomato
+-1.101,Initial Validation Unhandled Error,,,firebrick
+-2.101,Nifti Conversion Unhandled Error,,,teal
+-3.101,Brain Extract Unhandled Error,,,saddlebrown
+-4.101,Tumor Extract Unhandled Error,,,sandybrown
+-5.101,Manual Annotation Unhandled Error,,,peru
+-6.101,Label Segmentation Comparison Unhandled Error,,,moccasin
+-7.101,Annotation Confirmation Unhandled Error,,,mediumvioletred
```

Comment:
@aristizabal95 are you sure you want to upload this file to the main repo? It seems not a generalizeable IMO and relates just to one specific project. But I don't see any issues in comitting it here also